iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -352,6 +352,19 @@ static SmallVector<int64_t> getDefaultDistributedLevelTileSizes(
   }
   return distributedLevelTileSizes;
 }
+static SmallVector<int64_t> getDefaultDistributedLevelTileSizes(
+    linalg::LinalgOp linalgOp, ArrayRef<int64_t> minTileSizes,
+    ArrayRef<int64_t> maxTileSizes, ArrayRef<int64_t> vectorSizeHints = {}) {
+  OpBuilder builder(linalgOp.getContext());
+  builder.setInsertionPoint(linalgOp);
+  SmallVector<Range> iterationDomain =
+      linalgOp.createLoopRanges(builder, linalgOp.getLoc());
+  auto partitionableLoopsInterfaceOp =
+      cast<IREE::Flow::PartitionableLoopsInterface>(linalgOp.getOperation());
+  return getDefaultDistributedLevelTileSizes(
```

Comment:
I read getDefaultDistributedLevelTileSizes and it seems like it only cares about:

- Static start index of each dim (it seems to be always 0 from LingOp::createLoopRanges)
- Static size of each dim (kDynamicDim if dynamic)
- Set of indices that are partitionable
- Min/max tile sizes and vector size hints

So maybe it will be simpler to get these numbers here and just pass these static integers to getDefaultDistributedLevelTileSizes. WDYT?

Reply:
I agree that we only need `lbs`, `ubs`, `partitionableLoops`, `min/maxTileSizes`, and `vectorSizeHints`. For now, let's keep iterationDomain until we have unify methods in Linalg ops and LinalgExt ops.

I rework a bit for `partitionableLoops` logic. I make min/maxTileSizes 0 if they are not partitionable, this would make logic simpler. Please take a look, thanks!

Reply:
LGTM